### ISSUE TITLE
General tab content in Preferences Dialog may be too "accessibility" centric

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/joist#733]
This feedback came out of discussion in https://github.com/phetsims/joist/issues/724.

Current content is
![image](https://user-images.githubusercontent.com/6396244/125532424-f6f52a4f-ba2d-4bcb-97cb-14f94ecb156a.png)

It was mentioned that the Preferences Dialog will include things that are not related to accessibility, like sim options and a projector mode color profile. So this description could be make the Preferences Dialog seem more "accessibility" centric than it is.  Or perhaps this intro content should move to an "Accessibility" specific tab?

Assigning to @emily-phet and @terracoda to consider this feedback and decide if anything should change.

**Comment by emily-phet** [comment]
Thanks @jessegreenberg.

I do not interpret this first paragraph as accessibility specific. We intentionally use the phrasing "inclusive and accessible". "inclusive" is a general term, it's more inclusive to have sim-specific features to allow teachers to customize their use of the sim, etc. So "inclusive and accessible" is essentially like saying "flexible and accessible", or "customizable and accessible" if that helps. Because I use the term "inclusive design" as a description of our work and that's often associated with making things accessible, I can see how it might be interpreted by the team that "inclusive and accessible" are both referring to traditional accessibility features, but realistically few people know about "inclusive design" outside of tech design (or even inside of tech design!), and regardless, "inclusive design" is not specific to "accessibility features". 

The second paragraph we felt was important so that people who find the sim with minimal features in the preferences menu recognize there are sims with more features, particularly for those who may require certain features. 

Regarding the addition of an "accessibility" specific tab, a few things to note:
- There are plans for as many of 5 tabs already, based on different modality features planned.
- The role of a single "general" tab was decision of a group design process involving @kathy-phet @amanda-phet @arouinfar and others, that way all sims with preferences menus open to a consistent tab, so that when we demo or create PD materials this would be standard. There was a version designed when there were no sim-specific features (with the wording listed above), and one when there is sim specific features, as well as when there are other features on other tabs and when there's not. But the goal of having a "general" tab that the preferences menu always started with was intentional by the team. I'm not sure how an accessibility specific tab would work if added in addition to a general tab, or what language would then go in the general tab for a sim without sim-specific features. We circled around this issue as a team multiple times, and the current iteration is the solution we landed on.

My preference would be to leave it as is. 

As we often find in our public-facing materials that involve some traditional accessibility features as well as other new features, we have to bridge the language so that people recognize *both* the presence of general customizations, general customizations useful for people with certain disabilities, and of traditional accessibility features. I think of them all as inclusive features, but that language doesn't register the same for everyone so we tend to use a combination of "inclusive" and "accessible" so that people looking for one find it, and people who aren't looking for "accessible" are able to recognize that it's not *solely* accessibility features we're referring to.

**Comment by jessegreenberg** [comment]
During the meeting today @emily-phet and @terracoda brainstormed some different wording and came up with:

![image](https://user-images.githubusercontent.com/6396244/125677990-cd9bcfa4-0602-48fc-b18e-e5cd71198fe8.png)

@emily-phet and @terracoda will continue to brainstorm and update this issue.

**Comment by emily-phet** [comment]
In further discussion with @terracoda, we (mostly @terracoda!) came up with a second variant:

_We are adding features to our simulations to make them more inclusive. Explore the tabs in this menu to review or change the default presentation settings to support learners with diverse needs and within diverse environments._

_To find other simulations with accessibility features, go to the simulation filter page on the PhET website, and filter by Accessibility feature._ 

We liked that this was shorter, and got to an active ("explore") wording coupled with a reason why (...to support...), but also felt it was not really inclusive of all the purposes for the features that would ultimately go in Preferences menus. We thought that something more like "...to support teachers and learners with diverse needs, within diverse environments, and for diverse pedagogical goals") captured more comprehensively the span of features - including teacher/student, diverse need / diverse environment, and diverse pedagogies, but was too wordy and likely too challenging to find acceptably concise and understandable language.

**Comment by emily-phet** [comment]
The conclusion was to update the wording in the General tab of the Preferences menu to include:

_We are adding features to our simulations to make them more inclusive. Some of these features support accessibility for learners with diverse needs and within diverse environments. Explore the tabs in this menu to review or change the default presentation settings._

_To find other simulations with accessibility features, go to the simulation filter page on the PhET website, and filter by Accessibility feature._

We felt that this wording was:
- reasonably concise
- made more specific some of the intended meaning behind the world "inclusive", and 
- more specific and targeted in it's use of the word "accessibility".

Note - the wording for the General tab is challenging in part because of an explicit need to bridge the use of both "inclusive" and "accessibility. Here's why: we definitely wanted to use this space to let people know there are other sims with other features to investigate, and for multiple reasons the filtering option for those features is under the heading of "Accessibility". But, we actually conceptualize, and hope to engender greater awareness of, inclusion in the broadest sense of the term. We conceptualize "inclusive" as including but not *limited* to features solely intended for people with disabilities or solely for use of features by people with disabilities. We conceptualize features that support teachers and learners, use across locales, with different devices (i.e., mobile, laptop, desktop, etc.) and online/offline contexts, and to support different pedagogies and different pedagogical goals all as inclusive features.

**Comment by emily-phet** [comment]
@BLFiedler Since you've looked the most at survey feedback re preferences menu, can you take a look at the italicized text in my last comment, give a thumbs up if this change seems agreeable to you?

**Comment by emily-phet** [comment]
@jessegreenberg Please go ahead and implement the change to the wording as listed in italics here: https://github.com/phetsims/joist/issues/733#issuecomment-880167855

**Comment by jessegreenberg** [comment]
Updated, and can be seen here: https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-dev.45/phet/john-travoltage_en_phet.html

@emily-phet can you please review? If it looks OK can this be closed?

**Comment by emily-phet** [comment]
Looks great! Closing.
